src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -6,46 +6,49 @@
   >
     <v-radio
       id="radio-24"
-      class="radio-button pt-2"
+      class="pt-2"
+      color="#1976d2"
       label="2 years"
       value="24"
     />
     <v-radio
       id="radio-18"
-      class="radio-button pt-2"
+      color="#1976d2"
       label="18 months"
       value="18"
     />
     <v-radio
       id="radio-12"
-      class="radio-button pt-2"
+      color="#1976d2"
       label="12 months"
       value="12"
     />
     <v-radio
       id="radio-6"
-      class="radio-button pt-2"
+      color="#1976d2"
       label="6 months"
       value="6"
     />
-    <v-row class="ml-0 mt-0 radio-button pt-2">
+    <v-row class="ml-0 mt-0 pt-2">
       <v-radio
         id="radio-custom"
         class="mt-n4"
+        color="#1976d2"
         value="customMonths"
+        inline
       />
       <v-form ref="monthsRef">
         <v-text-field
           id="text-field-months"
           v-model="inputValue"
           class="text-field-months"
           type="number"
-          dense
-          filled
+          density="compact"
+          variant="filled"
           hide-spin-buttons
           :rules="monthsRules"
           :disabled="(radioValue !== 'customMonths')"
-          @input="onMonthsInput($event)"
+          @update:model-value="onMonthsInput($event)"
```

Comment:
Interesting. So this works? I wonder why it's different in Leo's component. Can you have a look there please?

Reply:
Sure.  I just left a comment on that PR